SirMorland/huld-hub
# feat: add search page

## Description
Added a search page.

### How to test
Run, login, and click search icon. Try resizing the page, the mock search bar and content of the page should always stay in the center and equal width.

### Test(s) added
No tests, just layout stuff.
